handbook/ce/team-culture.md

```diff
@@ -0,0 +1,180 @@
+
+--------
+
+## How we intersect other teams
+We work alongside just about every team at Sourcegraph. Here is how we add value to each other and a collaboration overview for each pairing.
+
+### Sales
+- **How CE adds value to sales:** CEs are the right-hand team for Sales by serving as technical experts; CEs lead product demos, guide customers through trials/POCs, lead production deployments, focuses on product usage and adoption.
+- **How sales adds value to CE:** Sales, with their focus on the commercial relationship, introduces CEs to the customer relationship and helps CEs nurture the relationship.
+- **Collaboration overview:** Paired together on accounts throughout the customer journey.
+
+### Customer Support Engineers (CSEs)
+- **How CE adds value to CSE:** CE has nuanced context that is valuable to how support works with a customer; CE can also help clarify / remind customers we need information (during regularly scheduled calls) on the more tricky issues.
+- **How CSE adds value to CE:** Support is the go-to technical team for our CEs. CSEs resolve issues for customers both pre- and post-sales, allowing CEs to do more proactive work by taking on the reactive technical troubleshooting work when customers experience issues.
+- **Collaboration overview:** CEs (or others -- including customers -- but primarily CEs) may engage support at any point during the pre-sales and post-sales customer engagement process. When a customer runs into an issue, the CE will allow support do the heads-down troubleshooting work.
+  - CEs may collaborate with CSEs on an issue. For example, if the CE has useful information or background that could be helpful in addressing the issue. Another example would be the CE has a suggested workaround to help alleviate the customer-reported issue. In scenarios such as these, if a CSE has already been assigned to an issue, the CE should engage with the CSE directly; to avoid unclear communication to the customer, the CE should directly engage the CSE rather than joining the customer thread. If the ticket has not yet been assigned, the CE can either let the customer know that the support team will be picking up their request and provide additional information that may be considered useful or the CE should provide the context in the support channel (via the Support Ticket Context shortcut) for when it is picked up by CSE. For each request the CE should provide the following information:
+    - Customer Name:
+    - Link to issue (slack / GH / ZD ticket #):
+    - Background information or anything we should know that would be helpful?:
+  - CE may also need to share additional context around an issue with the head of support. For example, a CE may need to share special circumstances around a customer issue. In scenarios such as this, the CE should raise a request in the support channel (via the Support Ticket Request shortcut).  For each request the CE should provide the following information:
```

Comment:
Do we document somewhere the Slack commands for raising this context?

Reply:
Yes above: (via the Support Ticket Context shortcut) and (via the Support Ticket Request shortcut).  Those are the names of the shortcuts